docs/how-to/cluster-cfg.rst

```diff
@@ -0,0 +1,60 @@
+Configure cluster_network/ public_network
+=========================================
+
+
+Overview
+--------
+Network configuration is critical for building a high performance Ceph Storage Cluster. Since, Ceph Clients make requests directly to Ceph OSD Daemons. Ceph OSD Daemons perform data replication on behalf of Ceph Clients, which means replication and other factors impose additional loads on Ceph Storage Cluster networks. Refer to `Ceph Network Config <https://docs.ceph.com/en/latest/rados/configuration/network-config-ref/>`_.
```

Comment:
This may need some clarification -- I'd suggest something like the below

----

Network configuration is critical for building a secure and well performing Ceph cluster. 

For resilience and performance reasons, dual NIC systems typically should implement active/active bonding or adopt a layer 3 multipath strategy to ensure maximum throughput and availability, and utilize a single network.

However keep in mind that clients are communicating directly with Ceph OSD daemons -- Ceph does not perform request routing. To enhance security and stability it can therefore be advantageous to split public and cluster network traffic so that client traffic flows on a public net while cluster traffic such as replication and backfilling utilizes a separate net. This helps to prevent malicious or malfunctioning clients from disrupting cluster backend operations.

Refer to `Ceph Network Config <https://docs.ceph.com/en/latest/rados/configuration/network-config-ref/>`_ for more detail.

Reply:
Replying to myself here -- this might be better suited for `Explanations` as well actually